test/src/unit-cppapi-deletes.cc

```diff
@@ -1455,4 +1503,40 @@ TEST_CASE_METHOD(
   CHECK(!memcmp(c_dim2.data(), dim2.data(), c_dim2.size() * sizeof(uint64_t)));
 
   remove_sparse_array();
-}
\ No newline at end of file
+}
+
+#ifndef _WIN32
```

Comment:
Remove?

Reply:
I tried but tiledb_vfs_copy_dir still doesn't work on windows.

Reply:
Ok, could you please add a comment so that the next person will know / hopefully be able to remove in the future.